src/app/modules/group/components/group-header/group-header.component.html

```diff
@@ -7,9 +7,14 @@
     </div>
     <ng-container *ngIf="navigationNeighbors$ | async as navigationNeighbors">
       <alg-page-navigator
-        *ngIf="!!groupWithManagement"
+        *ngIf="!!groupWithManagement && groupData"
```

Comment:
`groupWithManagement` is supposed to be `groupData` augmented with some management info... so `groupData` is not supposed to be used in the template and you are supposed to be able to use `groupWithManagement` everywhere you would use `groupData`

Reply:
Fixed